packages/integration-sdk-core/src/types/jobState.ts

```diff
@@ -100,10 +100,27 @@ export interface JobState {
    * steps it depends on. Other steps outside the dependency ancestry may not
    * have run and therefore entities collected by those other steps should not
    * be expected to exist.
+   *
+   * If concurrency is specified (defaults to 1), the iteratee will be executed
+   * concurrently for graph objects that are found in memory and for
+   * graph objects found in single graph file.
```

Comment:
nit: ```suggestion
   * graph objects found in graph files.
```
We usually store in groups of 250 graphObjects per file, there is not a single file per entity type

Reply:
Thanks, this wasn't clear. We don't concurrently execute across multiple files at a time. Only one file at a time is what I meant.